main.js

```diff
@@ -0,0 +1,39 @@
+const tweeter = Tweeter()
+const renderer = Renderer()
+let render = function () { renderer.renderPosts(tweeter.getPosts()) }
+
+const post = function () {
+    const inpval = $("#input").val()
+    tweeter.addPost(inpval)
+    render()
+}
+
+$("#posts").on('click', '.delete', function () {
+    const postid = $(this).closest(".post").data().id
+    tweeter.removePost(postid)
+    render()
+})
+
+$("#container").on("click", '.addcomment', function () {
+    const text = $(this).closest(".post").find(".commentinput").val()
```

Comment:
When you have more than one word for an HTML class/id, you should put dashes between them, for example: `comment-input`